RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate methods/loadNextMessages to typescript

## Proposed changes
Migrate methods/loadNextMessages to typescript

## Issue(s)	
N/A

## How to test or reproduce
Open a chat;
*  Navigate to old messages and copy the message link;
* Send the link in other chat, then tap it to verify if will load the messeges correctly;
* from there, check the oldest messages and then the newest ones;

Everything should work correctly

## Screenshots
N/A

## Types of changes




## Checklist



## Further comments
N/A
